src/main/java/org/drugis/addis/trialverse/service/TriplestoreService.java

```diff
@@ -68,5 +66,5 @@ static String loadResource(String filename) {
 
   List<URI> getUnitUris(String trialverseDatasetUuid, URI headVersion) throws IOException;
 
-  String getStudyTitle(String namespaceUid, URI versionUri, URI studyUri) throws IOException;
+  Map<URI, String> getStudyTitles(String namespaceUid, URI versionUri);
```

Comment:
our convention is to indicate what the keys are as well for map-returning functions
so in this case `getStudyTitlesByUri`